src/uct/cuda/cuda_copy/cuda_copy_md.c

```diff
@@ -202,14 +217,107 @@ UCS_PROFILE_FUNC(ucs_status_t, uct_cuda_copy_mem_dereg,
     return UCS_OK;
 }
 
-static ucs_status_t uct_cuda_copy_mem_alloc(uct_md_h md, size_t *length_p,
+static ucs_status_t
+uct_cuda_copy_mem_alloc_fabric(uct_cuda_copy_alloc_handle_t *alloc_handle)
+{
+#if HAVE_CUDA_FABRIC
+    CUdevice_attribute attrib   = CU_DEVICE_ATTRIBUTE_HANDLE_TYPE_FABRIC_SUPPORTED;
+    size_t granularity          = 0;
+    CUmemAllocationProp prop    = {};
+    CUmemAccessDesc access_desc = {};
```

Comment:
do we have to initialize?

Reply:
There are some internal members that we don't initialize explicitly and I've seen general usage of these variables being initialized in the above way. Can we we leave it initialized this way?